.gitignore

```diff
@@ -5,14 +5,16 @@
 *.tfstate
 *.tfstate.*
 
+# Terraform lock files
+.terraform.lock.hcl
```

Comment:
I think we should include

Reply:
The components are intended to be used as child modules, and the root module should have it's own lock file anyway. What's the reason we might want to keep it here?

Reply:
I think this component is already at root module level for a consumer. The most we would do is just copy it into `sts-devops`, no? to wrap it an additional time and write another `variable = var.variable ...` block seems like too much

Reply:
additionally I see no opinions online that lockfiles should not be included in child modules and don't see a reason not to.

Reply:
It's just not used by Terraform, no `init` is run on this level.

Reply:
> I think this component is already at root module level for a consumer. The most we would do is just copy it into `sts-devops`, no? to wrap it an additional time and write another `variable = var.variable ...` block seems like too much

Not sure. Let's discuss.